index.html

```diff
@@ -4,30 +4,40 @@
   <head>
     <title>Todo</title>
 
-    <meta charset="UTF-8">
-    <meta name="description" content="Very simple todo list">
-    <meta name="author" content="Shamim">
-    <meta name="viewport" content="width=device, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="description" content="Very simple todo list" />
+    <meta name="author" content="Shamim" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-    <link rel="stylesheet" href="./assests/css/index.css">
-    <script src="https://kit.fontawesome.com/ccf39592f4.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./assests/css/index.css" />
+    <script
+      src="https://kit.fontawesome.com/ccf39592f4.js"
+      crossorigin="anonymous"
+    ></script>
   </head>
 
   <body>
     <main class="main-box">
       <h1 class="title">Todo</h1>
+      <div id="connection-error" class="connection-error-empty-box"></div>
```

Comment:
Network error always has the same text. There is no need to fill the value from js here. You can just move those `p` tags here and hide/show it in case.

Reply:
thankssss